File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.js
let input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

let result = new Array(10).fill(0);
let multi = 1;

for (let i = 0; i < 3; i++) {
  multi = parseInt(input[i]) * multi;
}

for (let k = 0; k < multi.toString().length; k++) {
  for (let j = 0; j < 10; j++) {
    if (multi.toString()[k] == j) {
      result[j] += 1;
    }
  }
}

console.log(result.join(" "));

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.kt
fun main() {
    val A = readln().toInt()
    val B = readln().toInt()
    val C = readln().toInt()

    val result = mutableMapOf<Int, Int>()
    for (i in 0..9) {
        result[i] = 0
    }

    val num = (A * B * C).toString()

    val splitNum = num.split("").filter { it.isNotEmpty() }

    for (num in splitNum) {
        result[num.toInt()] = result[num.toInt()]!! + 1
    }

    for (i in 0..9) {
        println(result[i])
    }
}